### PR TITLE
Adding all the files for Capell 1956 558

### PR DESCRIPTION
**Pull request by MuffinLinwist**
# Pull request checklist

- [x] add new concept list
- [x] add new metadata
- [ ] add new Concepticon concept sets
  * [ ] checked whether the new concept(s) can be applied to existing lists with
    `concepticon notlinked --gloss "NEW_GLOSS"`
- [ ] add new Concepticon concept relations
- [ ] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information

...

### CODE REVIEW

**Comment by MuffinLinwist** [comment]
@LinguList and @lannin, here is the PR finished for the `Capell-1956-558` list.

@lannin, thanks for providing the TSV file with the list of mappings. In addition to this, each time, we also need to **update** the `concepticondata/conceptlist.tsv` file with a proper entry for the list. Next, is to create a **metadata JSON file** that specifies the structure of the concept list file, this corresponds to the `concepticondata/conceptlists/Capell-1956-558.tsv-metadata.json` file. This can be easily done with the `$ concepticon create_metadata` command. Finally, you add the **bib entry** in the `references` folder.

When all of this is finished, you'll have to do the **Concepticon check and test** to see if all is in order before making the PR. I did this and noticed there was some errors in the typing of some glosses and could effortlessly fix them.

If you forget some of this steps, you can always come back to this [blog post entry](https://calc.hypotheses.org/2225) from the project. It is always helpful! Any other question, you can come back to me :) 

@LinguList, now the update on `CONTRIBUTORS.md` is only missing. No one appears to be contributing with the 3.1 Concepticon release. Should I also add the information for the others?

File: concepticondata/conceptlists/Capell-1956-558.tsv [hunk 1]
```diff
@@ -0,0 +1,559 @@
...
+Capell-1956-558-93	1592	CHEST	93	chest	Capell1956[24]
+Capell-1956-558-94	1510	CHIN	94	chin	Capell1956[24]
+Capell-1956-558-95			95	chiton	Capell1956[24]
+Capell-1956-558-96	497	SCAR	96	cicatrix	Capell1956[24]
+Capell-1956-558-97	393	CIRCUMCISION	97	circumcision	Capell1956[25]
+Capell-1956-558-98	1399	CLAY	98	clay	Capell1956[25]
+Capell-1956-558-99	704	CLEAN	99	clean	Capell1956[25]
+Capell-1956-558-100	1310	CLEVER	100	clever	Capell1956[25]
+Capell-1956-558-101	1102	CLIMB	101	climb	Capell1956[26]
+Capell-1956-558-102	1046	SHUT	102	close (adj.)	Capell1956[26]
```
Comment (Tarotis) [inline concepticondata/conceptlists/Capell-1956-558.tsv:103]: SHUT refers to the action, but the entry seems to be quite specific about the adjectival/stative meaning

File: concepticondata/conceptlists/Capell-1956-558.tsv [hunk 2]
```diff
@@ -0,0 +1,559 @@
...
+Capell-1956-558-112			112	consider	Capell1956[28]
+Capell-1956-558-113	3179	CORAL ROCK	113	coral	Capell1956[29]
+Capell-1956-558-114			114	corroboree	Capell1956[29]
+Capell-1956-558-115	761	COVER	115	cover	Capell1956[29]
+Capell-1956-558-116	2348	CRAB	116	crab	Capell1956[29]
+Capell-1956-558-117	467	CRAYFISH	117	crayfish	Capell1956[30]
+Capell-1956-558-118	990	CREEK (TIDAL)	118	creek	Capell1956[30]
+Capell-1956-558-119	1857	CROCODILE	119	crocodile	Capell1956[30]
+Capell-1956-558-120	297	CROOKED	120	crooked	Capell1956[30]
+Capell-1956-558-121	1367	CROSS	121	cross (v.)	Capell1956[31]
```
Comment (Tarotis) [inline concepticondata/conceptlists/Capell-1956-558.tsv:122]: This should be TRAVERSE. I've opened an issue about this (#1316 ) as this has been mapped wrongly in some other cases as well.

File: concepticondata/conceptlists/Capell-1956-558.tsv [hunk 3]
```diff
@@ -0,0 +1,559 @@
...
+Capell-1956-558-193	235	FLYING FOX	193	flying fox	Capell1956[49]
+Capell-1956-558-194			194	flying phalanger	Capell1956[49]
+Capell-1956-558-195	249	FOG	195	fog	Capell1956[49]
+Capell-1956-558-196	256	FOOD	196	food	Capell1956[49]
+Capell-1956-558-197	1301	FOOT	197	foot	Capell1956[50]
+Capell-1956-558-198	123	FOREHEAD	198	forehead	Capell1956[50]
+Capell-1956-558-199	1523	FORGET	199	forget	Capell1956[50]
+Capell-1956-558-200	503	FROG	200	frog	Capell1956[50]
+Capell-1956-558-201	1429	FULL	201	full (of water)	Capell1956[51]
+Capell-1956-558-202	1429	FULL	202	full (of food)	Capell1956[51]
```
Comment (Tarotis) [inline concepticondata/conceptlists/Capell-1956-558.tsv:203]: These two should maybe be unmapped, as they are quite specific about their meaning, but the concept is not

File: concepticondata/conceptlists/Capell-1956-558.tsv [hunk 4]
```diff
@@ -0,0 +1,559 @@
...
+Capell-1956-558-215	1425	GREEN	215	green	Capell1956[54]
+Capell-1956-558-216			216	green ant	Capell1956[54]
+Capell-1956-558-217	479	GROW	217	grow	Capell1956[55]
+Capell-1956-558-218	139	GUMS	218	gum	Capell1956[55]
+Capell-1956-558-219	2648	HAIR (HEAD)	219	hair of head	Capell1956[55]
+Capell-1956-558-220			220	hairbelt	Capell1956[55]
+Capell-1956-558-221	1277	HAND	221	hand	Capell1956[56]
+Capell-1956-558-222	1495	HAPPY	222	happy	Capell1956[56]
+Capell-1956-558-223	1884	HARD	223	hard	Capell1956[56]
+Capell-1956-558-224	771	HAT	224	hat (dance)	Capell1956[56]
```
Comment (Tarotis) [inline concepticondata/conceptlists/Capell-1956-558.tsv:225]: Is this a specific hat? Then it should probably be unmapped

File: concepticondata/conceptlists/Capell-1956-558.tsv [hunk 5]
```diff
@@ -0,0 +1,559 @@
...
+Capell-1956-558-225	307	HAWK	225	hawk (chicken-h)	Capell1956[57]
```
Comment (Tarotis) [inline concepticondata/conceptlists/Capell-1956-558.tsv:226]: Should probably be unmapped, as it is a very specific type of hawk

File: concepticondata/conceptlists/Capell-1956-558.tsv [hunk 6]
```diff
@@ -0,0 +1,559 @@
...
+Capell-1956-558-254	1135	KICK	254	kick	Capell1956[64]
+Capell-1956-558-255	489	KIDNEY	255	kidney	Capell1956[64]
+Capell-1956-558-256	1417	KILL	256	kill	Capell1956[65]
+Capell-1956-558-257	2444	KINGFISHER	257	kingfisher	Capell1956[65]
+Capell-1956-558-258	1371	KNEE	258	knee	Capell1956[65]
+Capell-1956-558-259	1352	KNIFE	259	knife	Capell1956[65]
+Capell-1956-558-260			260	knick up	Capell1956[66]
+Capell-1956-558-261	3626	KNOW	261	know	Capell1956[66]
+Capell-1956-558-262			262	kulamin	Capell1956[66]
+Capell-1956-558-263	2444	KINGFISHER	263	kookaburra	Capell1956[66]
```
Comment (Tarotis) [inline concepticondata/conceptlists/Capell-1956-558.tsv:264]: As we have KINGFISHER mapped already, we should unmap this more specific entry

File: concepticondata/conceptlists/Capell-1956-558.tsv [hunk 7]
```diff
@@ -0,0 +1,559 @@
...
+Capell-1956-558-275	737	LIGHT (IGNITE)	275	light (a fire)	Capell1956[69]
+Capell-1956-558-276	1052	LIGHT (WEIGHT)	276	light (in weight)	Capell1956[70]
+Capell-1956-558-277	1654	LIGHTNING	277	lightning	Capell1956[70]
+Capell-1956-558-278			278	lily	Capell1956[70]
+Capell-1956-558-279			279	lily-root	Capell1956[70]
+Capell-1956-558-280	478	LIP	280	lip	Capell1956[71]
+Capell-1956-558-281	1608	LISTEN	281	listen	Capell1956[71]
+Capell-1956-558-282	1246	SMALL	282	little	Capell1956[71]
+Capell-1956-558-283			283	little while	Capell1956[71]
+Capell-1956-558-284	1422	BE ALIVE	284	live	Capell1956[72]
```
Comment (Tarotis) [inline concepticondata/conceptlists/Capell-1956-558.tsv:285]: This should probably be [2309 LIVE](https://concepticon.clld.org/parameters/2309)

File: concepticondata/conceptlists/Capell-1956-558.tsv [hunk 8]
```diff
@@ -0,0 +1,559 @@
...
+Capell-1956-558-472	797	TESTICLES	472	testicles	Capell1956[119]
+Capell-1956-558-473	800	THIGH	473	thigh	Capell1956[120]
+Capell-1956-558-474	2308	THIN	474	thin	Capell1956[120]
+Capell-1956-558-475	2271	THINK	475	think	Capell1956[120]
+Capell-1956-558-476	847	THIRSTY	476	thirsty	Capell1956[120]
+Capell-1956-558-477	1161	THREAD	477	thread	Capell1956[121]
+Capell-1956-558-478	492	THREE	478	three	Capell1956[121]
+Capell-1956-558-479	1346	THROAT	479	throat	Capell1956[121]
+Capell-1956-558-480	1456	THROW	480	throw	Capell1956[121]
+Capell-1956-558-481	135	BOOMERANG	481	throwing-stick	Capell1956[122]
```
Comment (Tarotis) [inline concepticondata/conceptlists/Capell-1956-558.tsv:482]: A throwing-stick does not necessarily be a boomerang, or can we be sure that it definitely refers to that?

File: concepticondata/conceptlists/Capell-1956-558.tsv [hunk 9]
```diff
@@ -0,0 +1,559 @@
...
+Capell-1956-558-545	1199	WIFE	545	wife	Capell1956[137]
+Capell-1956-558-546	960	WIND	546	wind	Capell1956[137]
+Capell-1956-558-547	962	WOMAN	547	woman	Capell1956[138]
+Capell-1956-558-548			548	wood (dry)	Capell1956[138]
+Capell-1956-558-549			549	wood (wet of green)	Capell1956[138]
+Capell-1956-558-550	1390	WRONG	550	wrong	Capell1956[138]
+Capell-1956-558-551	799	WRIST	551	wrist	Capell1956[139]
+Capell-1956-558-552	1257	WING	552	wing	Capell1956[139]
+Capell-1956-558-553	410	YAM	553	yam	Capell1956[139]
+Capell-1956-558-554	339	DIGGING STICK	554	yam-stick	Capell1956[139]
```
Comment (Tarotis) [inline concepticondata/conceptlists/Capell-1956-558.tsv:555]: Similarly to throwing stick and BOOMERANG, is a yam-stick always for digging? Probably an unmap

**Review by Tarotis — CHANGES_REQUESTED** [review]
Thanks for adding this list. I have a couple of comments (mostly unmaps) which should be taken into account before merging.

@AnnikaTjuka Could you do (or delegate) a final check once my requested changes are implemented?

File: concepticondata/conceptlists/Capell-1956-558.tsv (hunk 9 above)
Comment (MuffinLinwist) [reply concepticondata/conceptlists/Capell-1956-558.tsv:555]: I found definitions of digging stick on the internet that mentions yam stick as an alternative name for it. Do you want me to unmap it anyways?

Comment (Tarotis) [reply concepticondata/conceptlists/Capell-1956-558.tsv:555]: Then you can leave it, thanks for the research

**Review by Tarotis — APPROVED** [review]

**Comment by LinguList** [comment]
I'll try and do that tomorrow

File: concepticondata/conceptlists.tsv [hunk 10]
```diff
@@ -423,4 +423,5 @@ Hantgan-2022-348	Hantgan, Abbie and Babiker, Hiba and List, Johann-Mattis	2022
...
-Healey-1970-364	Healey, Alan	1970		364	basic	English	Awyu-Dumut Languages		Healey1970	Healey1970	This list contains concepts that are reconstructed for Awyu-Dumut languages.	1049-1058
\ No newline at end of file
+Capell-1956-558	Capell, Arthur	1956		558	basic	English	North Australian languages	https://indigenous.sl.nsw.gov.au/collection-items/vocabulary-600-words-forty-north-australian-languages-1956-arthur-capell	Capell1956	Capell1956	This concept list was taken from a book of vocabulary of 600 words in forty North Australian languages by [Capell (1956)](:bib:Capell1956). The data included fieldwork made by the linguist and anthropoligist with the back then recently contacted tribes of New Guinea.	1-140
```
Comment (LinguList) [inline concepticondata/conceptlists.tsv:426]: Reword from this:

> This concept list was taken from a book of vocabulary of 600 words in forty North Australian languages by [Capell (1956)](:bib:Capell1956). The data included fieldwork made by the linguist and anthropoligist with the back then recently contacted tribes of New Guinea.

to this:

> This concept list is taken from  [Capell's (1956)](:bib:Capell1956) study on 40 North Australian languages, based on original fieldwork.

**Review by LinguList — CHANGES_REQUESTED** [review]
I have methodological requests regarding the representation.

First, if you check the page numbers and the original source, you can see that we can in fact link to that source regularly:

```
https://indigenous.sl.nsw.gov.au/collection-items/vocabulary-600-words-forty-north-australian-languages-1956-arthur-capell#pid=6
```
The key part is `pid=` here, which means "page id -> 6". So we can use the page numbers which are by now hidden in the sources, to match the original entry, which is quite nice.

This means:

1. re-write SOURCE in the concept list by deleting the reference (which is superfluous) and keeping the page number in a column PAGE (which we use in other datasets)
2. duplicate the column PAGE and call the duplicated column URL
3. in the json file, add the following for the URL column:

```
                    {
                      "name": "URL",
                      "datatype": "string",
                      "propertyUrl": "dc:relation",
                      "valueUrl": "https://indigenous.sl.nsw.gov.au/collection-items/vocabulary-600-words-forty-north-australian-languages-1956-arthur-capell#pid={0}"
                    },
```

Finally, please find out and explain, why we have 558 concepts here, when the author mentions 600 in the title? This means, that you probably need to count items and make sure that none are missing. The easiest way to proceed here is to download the book images with wget or curl first...

**Comment by MuffinLinwist** [comment]
@LinguList, I have address the requests you made with the last commit of this PR. If everything is in order, we can proceed with merge.

**Pull request merged**